framework/src/modules/base_module.ts

```diff
@@ -26,7 +26,7 @@ import {
 } from '../types';
 import { BaseAsset } from './base_asset';
 
-interface Channel {
+export interface BaseModuleChannel {
```

Comment:
why exporting this type and changing the name here?

Reply:
It was needed to reference in a typescript file where I was creating some custom module. If you think we can remove the export. 

For name it was confusing when compared along with the second parameter of `BaseModule.init`

Reply:
yea let's remove `export` if not needed